### PR TITLE
VisibilityRequiredFixer - Add support for class const visibility added in PHP7.1.

### PR DESCRIPTION
**Pull request by SpacePossum**
for ref:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2090

### CODE REVIEW

File: src/Fixer/ClassNotation/VisibilityRequiredFixer.php [hunk 1]
```diff
@@ -21,15 +22,35 @@
...
 final class VisibilityRequiredFixer extends AbstractFixer
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function isCandidate(Tokens $tokens)
+    private $options = array('property', 'method', 'const');
+    private $configuration = array('property', 'method');
+
+    public function configure(array $configuration = null)
```
Comment (keradus) [inline src/Fixer/ClassNotation/VisibilityRequiredFixer.php]: Add documentation how to configure the fixer.

**Comment by keradus** [comment]
:+1: except minors

**Comment by SpacePossum** [comment]
note to self; maybe a test for single line multiple const declaring, ie:
```php
class A 
{
    const A = 1, B =2, C =3;
}
```

**Comment by keradus** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**